### PR TITLE
Finalize ads, add polyfill

### PR DESCRIPTION
**Pull request by combs**

### CODE REVIEW

File: h/ngmLeader-no-mobile-leaderboard.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+<script language="JavaScript1.1">
+
+var bust;
+if (typeof bust == 'undefined') {
+bust = Math.floor(1000000*Math.random());
+}
+ 
+if ( (!parent) || (parent.window.innerWidth > 1023)) {
+   if ((!document.images && navigator.userAgent.indexOf('Mozilla/2.') >= 0) || navigator.userAgent.indexOf("WebTV")>= 0) {
+   document.write('<a href="http://ad.doubleclick.net/jump/ng.ngm/ngmcontent;sz=728x90;tile=5;ord='+bust+'">');
```
Comment (chuckcarpenter) [inline h/ngmLeader-no-mobile-leaderboard.html]: document write is synchronous and VERY bad. Use something like this instead:
```javascript
        var script = document.createElement('script');

        script.async = true;
        script.src   = url;

        var entry = document.getElementsByTagName('script')[0];
        entry.parentNode.insertBefore(script, entry);
```

File: h/ngmLeader-no-mobile-leaderboard.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+   document.write('<img src="http://ad.doubleclick.net/ad/ng.ngm/ngmcontent;sz=728x90;tile=5;ord='+bust+'" border="0" height="90" width="728"></a>');
```
Comment (chuckcarpenter) [inline h/ngmLeader-no-mobile-leaderboard.html]: this is old DART style ads, we need to use GPT. https://developers.google.com/doubleclick-gpt/reference

Comment (combs) [inline h/ngmLeader-no-mobile-leaderboard.html]: Yeah--I was just making a small update to the DART logic. updating to GPT would definitely be best but I am balancing that against the fear of breaking everything in this crusty old system...



--
Chris CombsSenior Producer

National Geographic
+1.202.775.7850

Comment (combs) [inline h/ngmLeader-no-mobile-leaderboard.html]: p.s., I should have mentioned, the Cheetah templates pull this in as an iframe. So it should load async... I think. 

Perfect case would be to rip out the iframe and use GPT, but the iframe is locked away inside a template I can't reach.

**Pull request merged**